style.css

```
@@ -0,0 +1,150 @@
+body {
+    height: 100vh;
+    width: 100vw;
+    background-color: antiquewhite;
+    font-family: Georgia, 'Times New Roman', Times, serif;
+    margin: 0;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    overflow:visible;
+}
+.buttons{
+    display:flex;
+    flex-direction: row;
+    gap:10px;
+}
+.carousel {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    overflow: hidden;
+    width: 100%;
+    max-width: 60%;
+    height: auto;
+}
+
+
+.images {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+}
+
+.img {
+    box-shadow: 0px 4px 10px 0px grey;
+    border-radius: 15px;
+    width: 500px;
+    height: 300px;
+    object-fit: cover;
+    margin: 5px;
+}
+
+#back, #next {
+    height: 50px;
+    width: auto;
+    cursor: pointer;
+}
+
+.text {
+    overflow: hidden;
+    flex-direction: column;
+    transform: translateY(0px);
+    background-color: rgb(72, 11, 129);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border-radius: 30px;
+    padding: 15px;
+    color: white;
+    width: 90%;
+    max-width: 600px;
+}
+
+.text h3 {
+    margin: 10px 0;
+    font-size: 1.5rem;
+}
+
+.text p {
+    font-size: 1rem;
+    transition: font-size 0.2s;
+}
```

```suggestion
.text {
    h3 {
        margin: 10px 0;
        font-size: 1.5rem;
    }
    p {
        font-size: 1rem;
        transition: font-size 0.2s;
    }
}
```

Puede usar CSS Nesting